crates/runc/src/lib.rs

```diff
@@ -366,6 +366,7 @@ pub trait Spawner: Debug {
 /// and some other utilities.
 #[cfg(feature = "async")]
 impl Runc {
+    #[cfg(not(target_os = "linux"))]
```

Comment:
We don't have runc on non Linux environments. But also can apply same suggestion as above to avoid func duplication.